klaviyo/SFCC_Klaviyo
# Create Continuous Integration GitHub Jobs

## Description

- Adds base pre-commit, run across all files
  - Adds `ISML` linting
- Adds continuous integration
  - Uses changelogger, with all versioned files formatted to use changelogger
- Adds `changelogger` note to the PR template.

Addresses SFCC CI tickets:
  - TP163104
  - TP63106
  - TP165537


**NOTE** I've added tags to this project so we conform to semantic versioning. This allowed me to update the links properly without breaking any comparisons to previous versions. I left the old tags for historical purposes, but they tag the same version.

## Manual Testing Steps



1. Passing `pre-commit run --all-files`.
2. Passing `changelogger manage check`.

## Pre-Submission Checklist:

- [ ] You've updated the CHANGELOG following the steps [here](https://github.com/klaviyo/SFCC_Klaviyo#making-updates)
